docker-compose.yml

```diff
@@ -24,17 +16,16 @@ services:
             dockerfile: ./docker/django/Dockerfile
         command: bash -c 'tail -f /dev/null'
         environment:
-            DATABASE_URL: postgres://linkedevents:linkedevents@linkedevents-db/linkedevents
+            DATABASE_URL: postgis://linkedevents:linkedevents@linkedevents-db/linkedevents
         volumes:
-            - .:/code
-            - django-media-volume:/var/media/
+            - .:/app
+            - linkedevents-media:/var/media/
         ports:
             - "8000:8000"
         depends_on:
             - postgres
-            - elastic
         container_name: linkedevents-backend
 
 volumes:
-        postgres-data-volume:
-        django-media-volume:
+        linkedevents-postgres:
```

Comment:
docker prefixes all volumes with the folder's name, so the volume will be called `linkedevents_linkedevents-postgres`

Reply:
Argh. Forgot about that. I'll kick out the prefix. Is the `volume` postfix useful in some situations? Ie. are they listed in context where it is not immediately obvious they are volumes?

Reply:
I think `volume` postfix can be dropped.

Reply:
And Ville, if you could please pull latest changes before working again on commits, it'd be great as my Git global config was making my commits anonymous (which was intentional 😅) but I fixed it and had to do a force push (only and only this time) 👍

Reply:
All right. It seems to have confused git a wee bit, but nothing major.